src/hooks/useWeiConverter.ts

```diff
@@ -0,0 +1,24 @@
+import { useCurrencyConverter } from 'hooks/v1/CurrencyConverter'
```

Comment:
**suggestion** to follow the existing naming convention, we should name this file `WeiConverter.ts`

Reply:
Fo sho